examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,18 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(raw, low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
hmmm that's weird - it looks like the problem is with applying the projection. If I do proj=False, then the notches show up fine, while if I do proj=True, then they don't show up. I'm not sure what would be causing this. Could the act of applying the SSP be undoing the notch filtering somehow?